### PR TITLE
Feature: Community corporations

### PR DESCRIPTION
**Pull request by nwai90**
Introduce a new option for fan-made community corporations. OFF by default.

Screenshot 2020-09-20 at 4 20 27 PM

Screenshot 2020-09-20 at 3 53 18 PM

Screenshot 2020-09-25 at 12 21 04 AM

### CODE REVIEW

**Comment by alrusdi** [comment]
Please give me a moment to add russian translation to it :-)

**Comment by nwai90** [comment]
> Please give me a moment to add russian translation to it :-)

@alrusdi It's not a hurry :) Would be cool to iterate based on community feedback; there are a lot of strong players around. Maybe even a corporation design contest in future.

**Comment by kberg** [comment]
Should be easy enough for me to integrate this into #1324

File: src/cards/community/ProjectWorkshop.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    public play(player: Player) {
+        player.steel = 1;
+        player.titanium = 1;
+        return undefined;
+    }
+
+    public initialAction(player: Player, game: Game) {
+        player.cardsInHand.push(game.drawCardsByType(CardType.ACTIVE, 1)[0]);
+        
+        const drawnCard = game.getCardsInHandByType(player, CardType.ACTIVE).slice(-1)[0];
```
[the remark this answers is not in the document]
Comment (nwai90) [reply src/cards/community/ProjectWorkshop.ts]: 👍 It's cleaner that way; updated

**Review by alrusdi — APPROVED** [review]

**Review by vincentneko — APPROVED** [review]

**Pull request merged**